tests/integration/frameworks/wordpress/test_wordpress_do_action.php8.php

```diff
@@ -52,6 +52,10 @@ function do_action($tag, ...$args) {
     call_user_func_array($tag, $args);
 }
 
+//Simple mock of wordpress's get_theme_roots
+function get_theme_roots() {
+}
+
```

Comment:
How is this function used in the test?

Reply:
We are mocking wordpress here and forcing the framework detection to the wordpress.
In wordpress, we'll always try to call get_theme_roots, a wordpress provided function.
https://github.com/newrelic/newrelic-php-agent/blob/main/agent/fw_wordpress.c#L217-L229

Reply:
Could we make it a more complete mock like this:
```suggestion
//Simple mock of wordpress's get_theme_roots
function get_theme_roots() {
    return '/themes';
}
```
(based on https://github.com/WordPress/wordpress-develop/blob/6.1/src/wp-includes/theme.php#L380). I know the agent will be able to work with `get_theme_roots` that doesn't return anything useful but maybe it's worthwhile to test using something closer to real life. Or even better have three tests: 
1. do not return anything useful
2. return '/themes' string
3. return array of strings

Reply:
Yes, more mock could be added, but it's not part of this PR.
It was to have a function that the internal instrumentation could call without erroring.
The fw_wordpress logic handles if get_themes_roots doesn't return anything useful, so leaving one test like this and more that return other things helps exercise more code paths.